### PR TITLE
Iconography: informations

### PR DESCRIPTION
**Pull request by Michele-Camargo**
# What

Implement the new icons proposed in the [Iconography](https://www.figma.com/file/9WVVk8Vobz2ZMzmHjtEFEh/Iconography?node-id=1825%3A1490) spec

# Why

Enable us to have a greater visual collection, once we have a greater and more diverse amount of icons

# How

Added informations components in /components/Icons

# Sample

https://user-images.githubusercontent.com/44146877/166316580-a0846809-8c9d-49cb-bf6b-f39750e78a4a.mov

# QA

### CODE REVIEW

File: src/components/Icons/Information/InfoAddCircle.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
```
Comment (LopesAlysson) [inline src/components/Icons/Information/InfoAddCircle.tsx:6]: 🔧 Seria bom separar os imports de módulos dos relativos
```suggestion
import React from 'react';
import { Path } from 'react-native-svg';
import { colors } from '@magnetis/astro-galaxy-tokens';

import BaseIcon from '../BaseIcon';
import { getFill, getSize, getViewBox } from '../utils';
import type { IconProps } from '../types';
```

**Review by albertozaranza — APPROVED** [review]

File: src/components/Icons/Information/InfoBanCircle.tsx [hunk 2]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
```
Comment (LopesAlysson) [inline src/components/Icons/Information/InfoBanCircle.tsx:6]: 🔧 Mesma coisa aqui
```suggestion
import React from 'react';
import { Path } from 'react-native-svg';
import { colors } from '@magnetis/astro-galaxy-tokens';

import BaseIcon from '../BaseIcon';
import { getFill, getSize, getViewBox } from '../utils';
import type { IconProps } from '../types';
```

File: src/components/Icons/Information/InfoCheckCircle.tsx [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
```
Comment (LopesAlysson) [inline src/components/Icons/Information/InfoCheckCircle.tsx:6]: 🔧 Aqui também
```suggestion
import React from 'react';
import { Path } from 'react-native-svg';
import { colors } from '@magnetis/astro-galaxy-tokens';

import BaseIcon from '../BaseIcon';
import { getFill, getSize, getViewBox } from '../utils';
import type { IconProps } from '../types';
```

File: src/components/Icons/Information/InfoErrorCircle.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
```
Comment (LopesAlysson) [inline src/components/Icons/Information/InfoErrorCircle.tsx:6]: 🔧  Já sabe né, hehehe
```suggestion
import React from 'react';
import { Path } from 'react-native-svg';
import { colors } from '@magnetis/astro-galaxy-tokens';

import BaseIcon from '../BaseIcon';
import { getFill, getSize, getViewBox } from '../utils';
import type { IconProps } from '../types';
```

File: src/components/Icons/Information/InfoHelpCicle.tsx [hunk 5]
```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
```
Comment (LopesAlysson) [inline src/components/Icons/Information/InfoHelpCicle.tsx:6]: 🔧 
```suggestion
import React from 'react';
import { Path } from 'react-native-svg';
import { colors } from '@magnetis/astro-galaxy-tokens';

import BaseIcon from '../BaseIcon';
import { getFill, getSize, getViewBox } from '../utils';
import type { IconProps } from '../types';
```

File: src/components/Icons/Information/InfoInformationCicle.tsx [hunk 6]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
```
Comment (LopesAlysson) [inline src/components/Icons/Information/InfoInformationCicle.tsx:6]: 🔧 
```suggestion
import React from 'react';
import { Path } from 'react-native-svg';
import { colors } from '@magnetis/astro-galaxy-tokens';

import BaseIcon from '../BaseIcon';
import { getFill, getSize, getViewBox } from '../utils';
import type { IconProps } from '../types';
```

**Review by matheussilva2 — APPROVED** [review]

File: src/components/Icons/Information/InfoRemoveCircle.tsx [hunk 7]
```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
```
Comment (LopesAlysson) [inline src/components/Icons/Information/InfoRemoveCircle.tsx:6]: 🔧 
```suggestion
import React from 'react';
import { Path } from 'react-native-svg';
import { colors } from '@magnetis/astro-galaxy-tokens';

import BaseIcon from '../BaseIcon';
import { getFill, getSize, getViewBox } from '../utils';
import type { IconProps } from '../types';
```

File: src/components/Icons/Information/InfoWarning.tsx [hunk 8]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
```
Comment (LopesAlysson) [inline src/components/Icons/Information/InfoWarning.tsx:6]: 🔧 
```suggestion
import React from 'react';
import { Path } from 'react-native-svg';
import { colors } from '@magnetis/astro-galaxy-tokens';

import BaseIcon from '../BaseIcon';
import { getFill, getSize, getViewBox } from '../utils';
import type { IconProps } from '../types';
```

File: src/components/Icons/Information/InfoWarningCicle.tsx [hunk 9]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
```
Comment (LopesAlysson) [inline src/components/Icons/Information/InfoWarningCicle.tsx:6]: 🔧 
```suggestion
import React from 'react';
import { Path } from 'react-native-svg';
import { colors } from '@magnetis/astro-galaxy-tokens';

import BaseIcon from '../BaseIcon';
import { getFill, getSize, getViewBox } from '../utils';
import type { IconProps } from '../types';
```

**Review by matheussilva2 — APPROVED** [review]

File: jest.config.js [hunk 10]
```diff
@@ -21,6 +21,7 @@ module.exports = {
     '!src/components/Icons/Dashboard/**/*.{ts,tsx}',
     '!src/components/Icons/Support/**/*.{ts,tsx}',
     '!src/components/Icons/Arrows/**/*.{ts,tsx}',
+    '!src/components/Icons/Information/**/*.{ts,tsx}',
```
Comment (LopesAlysson) [inline jest.config.js:24]: ❓  Porque não estamos coletando coverage dessas pastas?

File: src/components/Icons/Information/InfoErrorCircle.tsx [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
+
+function InfoErrorCircle({
+  id = 'InfoErrorCircleIcon',
+  color = colors.space100,
+  viewBox = getViewBox(),
```
Comment (pedro-cerdera) [inline src/components/Icons/Information/InfoErrorCircle.tsx:11]: ❓ 
Alterar o ViewBox manualmente não pode ser perigoso por conta do ícone ficar cortado?  Sabendo disso, faz sentido estar disponível?

**Review by pedro-cerdera — APPROVED** [review]

File: src/components/Icons/Information/InfoErrorCircle.tsx (hunk 11 above)
Comment (pedro-cerdera) [reply src/components/Icons/Information/InfoErrorCircle.tsx:11]: Vou trazer o ponto na reunião do astro

File: src/components/Icons/Information/InfoAddCircle.tsx (hunk 1 above)
Comment (Michele-Camargo) [reply src/components/Icons/Information/InfoAddCircle.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/565/commits/7430dbdeb25fa06af4dcf3f8522f4cd0caddf050

File: src/components/Icons/Information/InfoBanCircle.tsx (hunk 2 above)
Comment (Michele-Camargo) [reply src/components/Icons/Information/InfoBanCircle.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/565/commits/7430dbdeb25fa06af4dcf3f8522f4cd0caddf050

File: src/components/Icons/Information/InfoCheckCircle.tsx (hunk 3 above)
Comment (Michele-Camargo) [reply src/components/Icons/Information/InfoCheckCircle.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/565/commits/7430dbdeb25fa06af4dcf3f8522f4cd0caddf050

File: src/components/Icons/Information/InfoErrorCircle.tsx (hunk 4 above)
Comment (Michele-Camargo) [reply src/components/Icons/Information/InfoErrorCircle.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/565/commits/7430dbdeb25fa06af4dcf3f8522f4cd0caddf050

File: src/components/Icons/Information/InfoInformationCicle.tsx (hunk 6 above)
Comment (Michele-Camargo) [reply src/components/Icons/Information/InfoInformationCicle.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/565/commits/7430dbdeb25fa06af4dcf3f8522f4cd0caddf050

File: src/components/Icons/Information/InfoRemoveCircle.tsx (hunk 7 above)
Comment (Michele-Camargo) [reply src/components/Icons/Information/InfoRemoveCircle.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/565/commits/7430dbdeb25fa06af4dcf3f8522f4cd0caddf050

File: src/components/Icons/Information/InfoWarning.tsx (hunk 8 above)
Comment (Michele-Camargo) [reply src/components/Icons/Information/InfoWarning.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/565/commits/7430dbdeb25fa06af4dcf3f8522f4cd0caddf050

File: src/components/Icons/Information/InfoWarningCicle.tsx (hunk 9 above)
Comment (Michele-Camargo) [reply src/components/Icons/Information/InfoWarningCicle.tsx]: comment resolved in https://github.com/magnetis/astro-native/pull/565/commits/7430dbdeb25fa06af4dcf3f8522f4cd0caddf050

**Review by lucianomlima — APPROVED** [review]

File: jest.config.js (hunk 10 above)
Comment (Michele-Camargo) [reply jest.config.js:24]: Pois os ícones usam o `BaseIcon` e o `BaseIcon` ja é testado, então seria redundante testar novamente. Dessa forma fazemos com que o Coverage pule esses arquivos na hora de testar.

File: src/components/Icons/Information/InfoAddCircle.tsx [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+
+function InfoAddCircle({
+  id = 'InfoAddCircleIcon',
+  color = colors.space100,
+  viewBox = getViewBox(),
+  width = getSize(),
+  height = getSize(),
+  ...props
+}: IconProps) {
+  const fill = React.useMemo(() => getFill({ gradient: props.gradient, color, id }), [color, props.gradient, id]);
```
[the remark this answers is not in the document]
Comment (Michele-Camargo) [reply src/components/Icons/Information/InfoAddCircle.tsx:17]: We had gradient icons that changed after rendering and we had performance issues. Not necessary today, but we will prioritize this refactoring in the future

**Pull request merged**